<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarEstacion</class>
 <widget class="QWidget" name="AgregarEstacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Estación</string>
  </property>
  
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Estación</string>
   </property>
  </widget>

  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>  
   </property>
  </widget>
  
  <widget class="QLineEdit" name="line_edit_nombre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="label_linea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Línea:</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="line_edit_linea">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="label_direccion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirección:</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="line_edit_direccion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>

  <widget class="QPushButton" name="btn_guardar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
